frontend/src/metabase/admin/settings/components/widgets/EmbeddingSwitchWidget/EmbeddingSwitchWidget.tsx

```diff
@@ -0,0 +1,31 @@
+import { t } from "ttag";
+import * as MetabaseAnalytics from "metabase/lib/analytics";
+import { Stack, Switch, Text } from "metabase/ui";
+
+interface EmbeddingSwitchWidgetProps {
+  setting: { value: boolean | null };
+  onChange: (value: boolean) => void;
+}
+
+export const EmbeddingSwitchWidget = ({
+  setting,
+  onChange,
+}: EmbeddingSwitchWidgetProps) => (
+  <Stack spacing={"md"} className="text-measure">
+    <Text lh={1.5}>
+      {t`Embed dashboards, questions, or the entire Metabase app into your application. Integrate with your server code to create a secure environment, limited to specific users or organizations.`}
+    </Text>
+    <Switch
+      labelPosition="left"
+      checked={setting.value ?? false}
+      label={<strong>{t`Embedding Enabled`}</strong>}
+      onChange={e => {
+        onChange(e.currentTarget.checked);
+        MetabaseAnalytics.trackStructEvent(
```

Comment:
Previously this was only called when going from disabled -> enabled.
I'll ask product and change accordingly